com.vrcfury.vrcfury/Editor/VF/Hooks/Av3EmuAnimatorFixHook.cs

```diff
@@ -57,6 +61,18 @@ internal class Av3EmuAnimatorFixHook : IVRCSDKPreprocessAvatarCallback {
                 var runHooksField =
                     av3EmulatorType.GetField("RunPreprocessAvatarHook", BindingFlags.Instance | BindingFlags.Public);
                 if (runHooksField == null) throw new Exception("Failed to find RunPreprocessAvatarHook field");
+                    

```

Comment:
This should probably happen only if there's at least one thing in `emulators`, just to make sure we're not touching things unnecessarily when av3emu isn't actually running.

Reply:
Fair enough, fixed